docs/use-signalr-binding.md

```diff
@@ -0,0 +1,747 @@
+# Azure SignalR Signalr Service Binding Usage <!-- omit in toc -->
+
+## Contents <!-- omit in toc -->
+- [Prerequisites](#prerequisites)
+- [Usage](#usage)
+  - [Create Azure SignalR Service instance](#create-azure-signalr-service-instance)
+  - [Create Function App with extension](#create-function-app-with-extension)
+  - [Add application settings](#add-application-settings)
+  - [Use input binding](#use-input-binding)
+    - [2.x C# input examples2.x C# input examples](#2x-c-input-examples2x-c-input-examples)
+      - [Authenticated token](#authenticated-token)
+    - [2.x JavaScript input examples](#2x-javascript-input-examples)
+      - [Authenticated token](#authenticated-token-1)
+    - [2.x Java input examples](#2x-java-input-examples)
+      - [Authenticated token](#authenticated-token-2)
+  - [Use output binding](#use-output-binding)
+    - [2.x C# output samples](#2x-c-output-samples)
+      - [2.x C# send message output examples](#2x-c-send-message-output-examples)
+      - [2.x C# group management output examples](#2x-c-group-management-output-examples)
+      - [Use SignalR Service Management SDK in functions](#use-signalr-service-management-sdk-in-functions)
+    - [2.x JavaScript output examples](#2x-javascript-output-examples)
+      - [2.x JavaScript send message output examples](#2x-javascript-send-message-output-examples)
+      - [2.x JavaScript group management output examples](#2x-javascript-group-management-output-examples)
+    - [2.x Java output examples](#2x-java-output-examples)
```

Comment:
note that java binding is still in v1.0.0, dosen't support connection-related and RemoveUserFromAllGroups features